### PR TITLE
Correct document and add TSPlayer property for TeleportEventArgs

### PR DESCRIPTION
**Pull request by mistzzt**

### CODE REVIEW

File: TShockAPI/GetDataHandlers.cs [hunk 1]
```diff
@@ -1159,12 +1159,16 @@ private static bool OnPlayerBuffUpdate(byte id)
...
 		/// </summary>
 		public class TeleportEventArgs : HandledEventArgs
 		{
 			/// <summary>
-			/// ???
+			/// The player who teleported
+			/// </summary>
+			public TSPlayer Player { get; set; }
+			/// <summary>
+			/// ID of the target player
```
Comment (WhiteXZ) [inline TShockAPI/GetDataHandlers.cs]: Is this the ID of the TSPlayer object?

Comment (mistzzt) [inline TShockAPI/GetDataHandlers.cs]: ID is the target player's id.
Player is the one who teleported using rod of discord or wormhole potion.

Comment (WhiteXZ) [inline TShockAPI/GetDataHandlers.cs]: So when using a wormhole potion, ID is the player being teleported *to*,
and when using a RoD, ID should be the same as Player.ID?

Comment (mistzzt) [inline TShockAPI/GetDataHandlers.cs]: After reading terraria's source code, I found that the id is always the player who teleported.
And server use the distance between x,y and players to find the player being teleported to.

So, i made a mistake

**Comment by mistzzt** [comment]
Closed as the Player property is actually not needed.

**Pull request closed without merging**